python/ambassador/config/resourcefetcher.py

```diff
@@ -852,12 +852,14 @@ def handle_k8s_service(self, k8s_object: AnyDict) -> HandlerResult:
                 self.filename += ":annotation"
 
             try:
-                objects = parse_yaml(annotations, namespace=resource_namespace)
```

Comment:
Passing in the `resource_namespace` here was just an illusion. The receiving method made no use of it.